### ISSUE TITLE
allow multiple email addresses

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#1450]
AMY currently allows us only one email address per individual.  It would be good to have at least two - identified as primary and secondary so we have other ways to contact individuals.

**Comment by maneesha** [comment]
@Talishask @sheraaronhurt 
This has come up when we've tried to contact an individual but they were no longer using the email address we had on record, or were on temporary leave, or something like that.

If we did this, how many email addresses would you want to see defined? How would you want to categorize them?  Should the individual designate which one they want to be the primary/seconday?

**Which of the following applies to when you want to see this fixed?**
1. Would like to see this fixed as soon as possible, in the next 1-2 development cycles (i.e., by September 2019)
2. Would like to see this fixed later (by the end of 2019)
3. Would like to see this fixed, but possibly as part of a larger project that needs more planning
4. Not sure
5. No longer relevant - OK to close

**Comment by Talishask** [comment]
1 or 2 - Most of us have multiple emails (work, school, personal, etc.). Primary and Secondary are great designations because it doesn't put a label on the email address that can be shared and also gives us useful information as to user preferences. It would be great to allow cc/multiple emails in the workshop request form and also in the user information (the person information on AMY). We've been using emails as the unique identifier for instructors/helpers, but at times instructors have used different emails.

**Comment by sheraaronhurt** [comment]
agreed 1 or 2

**Comment by maneesha** [comment]
This should rename the current email field as "Primary email" and then add a field called "Secondary email."  Once this is in place, we will reach out to instructors and invite them to update their profiles with a secondary email address.

We should not call these emails "work email" or "personal email" or anything like that as it is up to the individual which they want to use as primary and which as secondary.
